src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Views/Controls/SampleChooserControl.xaml

```diff
@@ -340,7 +340,7 @@
 
 				<!--NAVIGATION BUTTONS-->
 				<Border Background="{StaticResource Color08Brush}"
-						Grid.ColumnSpan="2"
```

Comment:
Why is this needed ?

Reply:
To be able to test at some point I had to prevent any overflow. In this case, there's only one column, so the ColumnSpan were crashing my first version of the Grid. That's fixed now. Do you think I should revert this, since it's pratically a non-sense?